app/src/main/java/ch/epfl/qedit/view/edit/EditOverviewFragment.java

```diff
@@ -11,21 +11,22 @@
 import androidx.fragment.app.Fragment;
 import androidx.lifecycle.ViewModelProvider;
 import ch.epfl.qedit.R;
+import ch.epfl.qedit.Search.StringSearchable;
 import ch.epfl.qedit.model.Question;
 import ch.epfl.qedit.model.StringPool;
 import ch.epfl.qedit.view.util.ListEditView;
 import ch.epfl.qedit.viewmodel.EditionViewModel;
 import java.util.ArrayList;
-import java.util.List;
 
 /** This fragment is used to view and edit the list of questions of a quiz. */
 public class EditOverviewFragment extends Fragment {
+    private StringSearchable titles = new StringSearchable();;
     public static final String QUESTION = "ch.epfl.qedit.view.edit.QUESTION";
     public static final int EDIT_QUESTION_ACTIVITY_REQUEST_CODE = 0;
-
-    private ListEditView.Adapter<String> adapter;
+    private ListEditView.Adapter<String, StringSearchable> adapter;
```

Comment:
Is it the goal to be able to search in the edition workflow? Otherwise I do not understand why this part should be affected by the search code.